src/Mono.Android/Android.Runtime/JNIEnvInit.cs

```diff
@@ -32,13 +32,15 @@ internal struct JnienvInitializeArgs {
 			public bool            jniRemappingInUse;
 			public bool            marshalMethodsEnabled;
 			public IntPtr          grefGCUserPeerable;
+			public bool            managedMarshalMethodsLookupEnabled;
 		}
 #pragma warning restore 0649
 
 		internal static JniRuntime.JniValueManager? ValueManager;
 		internal static bool IsRunningOnDesktop;
 		internal static bool jniRemappingInUse;
 		internal static bool MarshalMethodsEnabled;
+		internal static bool ManagedMarshalMethodsLookupEnabled;
```

Comment:
Given that this field isn't (yet) used by anything, shouldn't we *skip* this field?  It'll reduce IL size, if nothing else.

Reply:
I think this is a leftover from when I started by copying existing propagation of the `MarshalMethodsEnabled` flag and I did not go back and remove the unused ones.